test/integration/addons_test.go

```diff
@@ -58,7 +58,7 @@ func TestAddons(t *testing.T) {
 		t.Fatalf("Failed setting GOOGLE_CLOUD_PROJECT env var: %v", err)
 	}
 
-	args := append([]string{"start", "-p", profile, "--wait=true", "--memory=4000", "--alsologtostderr", "--addons=registry", "--addons=metrics-server", "--addons=olm", "--addons=volumesnapshots", "--addons=csi-hostpath-driver", "--addons=gcp-auth"}, StartArgs()...)
+	args := append([]string{"start", "-p", profile, "--wait=true", "--memory=4000", "--alsologtostderr", "--force", "--addons=registry", "--addons=metrics-server", "--addons=olm", "--addons=volumesnapshots", "--addons=csi-hostpath-driver", "--addons=gcp-auth"}, StartArgs()...)
```

Comment:
why did we have to do force in integration test ? 
if really needed comment why we are doing this.
I dont like that our integration test tests with --force. if possible try not using it

Reply:
the integration tests run in GCE

Reply:
instead of that, how about checking is isGCE and then valdiate if minikube did the rigth thing.
lets not add --force here
and we could add a full test for GCP auth addon as a separate test, since it is more important addon we could do all kind of tests on it

if isGCE then expect it skips the creds
if not and nil then does this...

so lets not add --force as our own code thats sets a a bad example

Reply:
if we want to test gcp auth in any capacity in GCE, we're going to need the force flag by design, we can break it out into its own test, but it won't change the fact that we'd need to use it.

Reply:
alright, I moved some code around and added comments to make it clear that --force shouldn't normally be used, and i check the error message if we don't supply --force in GCE.